schema/buildings/defs.yaml

```diff
@@ -20,6 +20,12 @@ shapeContainer:
       description: >-
         The height of the bottom part of building in meters. Used if a building or part of building starts above the ground level.
       type: number
+    min_floor:
+      description: >-
+        The "start" floor of this building or part. This is the analog of "min_height" but is sometimes populated when the
+        start height is unknown.
```

Comment:
Should we update this description a bit based on what we learned looking at the docs page in schema TF meeting the other day?

It seems to serve a different purpose than `min_height`, although admittedly in the rendering use case you can presumably use `min_floor` as a proxy for `min_height`...

Maybe along the lines of:

> The "start" floor of this building or part. Indicates that the building or part is "floating" and its bottom-most floor is above ground level, usually because it is part of a larger building in which some parts do reach down to ground level. An example is a building that has an entry road or driveway at ground level into an interior courtyard, where part of the building bridges above the entry road. This property may sometimes be populated when `min_height` is missing and in these cases can be used as a proxy for `min_height`.